modules/toast/react/spec/Toast.spec.tsx

```diff
@@ -41,8 +41,8 @@ describe('Toast', () => {
           {toastMessage}
         </Toast>
       );
-      const closeIcon = container.querySelector('[data-close="close"]'); /*? */
-      fireEvent.click(closeIcon); /*? */
```

Comment:
this was some  comments that where left, seemed minor enough to remove

Reply:
I think these are also removed in v4